CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change history for ui-notes
 
+## [5.0.0] (IN PROGRESS)
+
+* Increment `notes` interface to `2.0`
+
```

Comment:
gotta actually update the version string in `package.json` then too :)

Reply:
Oh, I thought we were supposed to do this only in `Release vX.Y.Z` commits. I'll update it here then

Reply:
TL;DR: bump the minor on master after each "Release vM.m.0" commit, e.g. if you release `v3.0.0` bump it to `3.1.0` and if you release `3.8.0` bump it to `3.9.0`. 

The details: Versioning is messy. If you don't bump the version on master after a major or minor release, patch releases have the potential to confuse the heck out of workspaces. Imagine the following: 

We release some library with version `3.0.0` and leave the version master alone, then commit a bunch of stuff, then decide to publish one of those commits in a patch release. We'll cherry-picking that commit from master (where, without a version bump, the version would still be `3.0.0`) onto a release branch (where it will be released as `3.0.1`) and then an app updates its dep from `^3.0.0` to `^3.0.1`. That's all fine, but here's the problem: if you have a workspace with that library and that app and you git-pull the app in your workspace, its `^3.0.1` dep is unsatisfied by the copy of that library that is cloned into your workspace. If it's a peer-dep (which `@folio/stripes` is for all apps), yarn _should_ just show you an unsatisfied-peers warning. But sometimes yarn gets eager and wants to "help" and pulls the `3.0.1` copy into your node_modules, or maybe another app on your platform misconfigured that lib as a direct dep instead of a peer. 

Point is, there are lots of easy ways for humans and robots to screw things up with versions. Bumping the minor after a major or minor release can make the consequences of those screw-ups less painful.

Reply:
Also, I'm super unreliable about handling all the version bumps. Like, super bad. Do not worry about having missed this. As above, there are lots of easy ways for humans to screw things up with versions. 😅